contrib/hyperscan/matching/input_matchers/source/BUILD

```diff
@@ -26,6 +26,7 @@ envoy_cmake(
         "BUILD_AVX512": "on",
         "BUILD_AVX512VBMI": "on",
         "BUILD_EXAMPLES": "off",
+        "CMAKE_BUILD_TYPE": "Release",
```

Comment:
i think you will need to add the hack to override the default here - add:

```starlark
default_cache_entries = {}
```
to the cmake rule

Reply:
Yeah, forgot to clean up default entries.